config/_shared.json

```diff
@@ -1,6 +1,7 @@
 {
 	"env": "shared",
 	"env_id": "shared",
+	"favicon_url": "//s1.wp.com/i/favicon.ico",
```

Comment:
Shouldn't we also add an explicit `favicon_url` to `config/production.json`?

Reply:
Not sure. Previously it was the fallback/default as well. I don't mind adding it.

Reply:
Yeah, let's please add it. Personally, when I find a config setting in all the other ones, I'd go looking for the production setting in `production.json`.

Reply:
done